src/autodiscovery.js

```diff
@@ -18,7 +18,7 @@ limitations under the License.
 
 import Promise from 'bluebird';
 const logger = require("./logger");
-import { parseURL } from "whatwg-url";
+import { URL as NodeURL } from "url";
```

Comment:
No need to import it, URL is available on the global object in node, just like in the browser.

Reply:
It's not, I tried and it yelled at me.